examples/ants.jl

```diff
@@ -447,11 +441,15 @@ data, _ = run!(
 
 rename!(data, [:step, :amount_cache_1, :amount_cache_2, :amount_cache_3]);
 
-# wide to long format
-data_melt = stack(
-    data, 
-    [:amount_cache_1, :amount_cache_2, :amount_cache_3], [:step]; 
-    variable_name=:cache_number, value_name=:amount);
+function plot_food_timeseries(data)
+    figure = Figure(resolution = (1200, 800))
+    ax = figure[1, 1] = Axis(figure; xlabel = "Step", ylabel = "Food in cache")
+    c1 = lines!(ax, data.step, data.amount_cache_1, color = :blue)
+    c2 = lines!(ax, data.step, data.amount_cache_2, color = :orange)
+    c3 = lines!(ax, data.step, data.amount_cache_3, color = :green)
+    figure[1, 2] = Legend(figure, [c1, c2, c3], ["Cache 1", "Cache 2", "Cache 3"])
+    return figure
+end
 
-p = ggplot(data_melt, x=:step, y=:amount, color=:cache_number, Geom.line)
-p|> SVG("cache_amounts.svg")
\ No newline at end of file
+fig = plot_food_timeseries(data);
+save("cache_amounts.png", fig);
```

Comment:
There is no reason to save the figure, having the figure as final argument already shows it at the built docs.

Reply:
hmmm, what about the video? should it be saved?

Reply:
ah wait thats saved by default nvm

Reply:
I removed the saving of that plot